examples/operator/templates/deployment.yaml

```diff
@@ -91,7 +91,20 @@ spec:
         resources: {{- toYaml .Values.controllerManager.manager.resources | nindent 10
           }}
         securityContext:
-          allowPrivilegeEscalation: false
+          allowPrivilegeEscalation: {{ .Values.controllerManager.manager.containerSecurityContext.allowPrivilegeEscalation
```

Comment:
I think that we don't have to template each field individually here. It is more readable to render the whole yaml for securityContext from values.yaml. In the same way, as it is done above for resources. Or for [nodeSelector here](https://github.com/arttor/helmify/commit/db73ef44705129b0663316226ca860865cd3fa74)

Reply:
done! :)